gateway/bucketfs.go

```diff
@@ -72,8 +72,9 @@ func (f *bucketFileSystem) Exists(bucket, pth string) (bool, string) {
 	ctx, cancel := context.WithTimeout(context.Background(), f.timeout)
 	defer cancel()
 	rep, err := f.client.ListBucketPath(ctx, "", path.Join(bucket, pth), f.auth)
-	if err == nil && !rep.Item.IsDir {
-		return true, ""
+	if err != nil {
+		log.Errorf("error listing bucket path: %v", err)
```

Comment:
not confident that this log is actually useful... thoughts?

Reply:
Why's that? It's good practice to log an error if you're not returning it.

Reply:
That is to say, looks like a nice addition.

Reply:
cool, all good then